es-app/src/views/gamelist/ISimpleGameListView.cpp

```diff
@@ -160,6 +161,15 @@ bool ISimpleGameListView::input(InputConfig* config, Input input)
 		}
 	}
 
+	FileData* cursor = getCursor();
+	SystemData* system = this->mRoot->getSystem();
+    	if (system != NULL) {
+            Scripting::fireEvent("game-select", system->getName(), cursor->getFileName(), cursor->getName(), "input");
```

Comment:
Consider using cursor->getSystemName() here instead of system->getName(). for game lists like favorites to receive the actual system name instead of "favorites"

Reply:
Actually that is the behavior I was looking for.  For example, if someone is in a custom collection e.g "Aliens" and they have a ROM that doesn't have artwork for the pixelcade specifically for it, this code will tell it to display the "Aliens" system artwork.  The idea being the default shows where you are, not system the game is from.